doc/API.md

```diff
@@ -182,6 +182,7 @@ This command will refuse to create any output worth less than 5k sats.
 | Field          | Type      | Description                                          |
 | -------------- | --------- | ---------------------------------------------------- |
 | `psbt`         | string    | PSBT of the spending transaction, encoded as base64. |
+| `warnings`     | list      | Warnings, if any, generated during spend creation.   |
```

Comment:
Might it be useful for the GUI to have access to the enum value, e.g. if the GUI needs to parse the warning to extract a value, e.g. to create a link to a transaction using its txid, it would be simpler with the enum.

Reply:
The GUI needs to have access to the same interface when consuming the JSONRPC interface or directly using the Rust functions.